pkg/cmd/attestation/verification/extensions.go

```diff
@@ -13,25 +13,35 @@ var (
 	GitHubTenantOIDCIssuer = "https://token.actions.%s.ghe.com"
 )
 
-func VerifyCertExtensions(results []*AttestationProcessingResult, ec EnforcementCriteria) error {
+func VerifyCertExtensions(results []*AttestationProcessingResult, ec EnforcementCriteria) ([]*AttestationProcessingResult, error) {
 	if len(results) == 0 {
-		return errors.New("no attestations proccessing results")
+		return nil, errors.New("no attestations processing results")
 	}
 
+	verified := make([]*AttestationProcessingResult, len(results))
```

Comment:
Reading over https://go.dev/blog/slices-intro, i think you can be like:

```suggestion
	verified := make([]*AttestationProcessingResult, 0, len(results))
```